tools/custom/raw-bytecode-analyser/raw_bytecode_analyser.py

```diff
@@ -0,0 +1,55 @@
+from evmdasm import EvmBytecode
+import argparse
+import requests
+
+def fetch_contract_bytecode(mirror_node_url, contract_id):
+    url = f'{mirror_node_url}/api/v1/contracts/{contract_id}'
+    response = requests.get(url)
+
+    if response.status_code == 200:
+        bytecode = response.json().get('runtime_bytecode')
+        if bytecode:
+            address_detected = False
+            method_detected = False
+            param_detected = False
+            call_detected = False
+
+            evm_bytecode = EvmBytecode(bytecode)
+            disassembled_code = evm_bytecode.disassemble()
+            for instruction in disassembled_code:
+                if instruction.operand == '0167':
```

Comment:
I don't know the library `evmdasm` but why isn't this `instruction.address`?

Reply:
@david-bakin-sl instruction.address represents Opcode Value, which uniquely identifies the operation to be performed. This is not the address of the Smart Contract's we are calling.
@acuarica yes, and we can ignore usage of these values in JUMP/DUP/SWAP/LOG in this context.